Yoast/wordpress-seo
# DUPP-138 add cards

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Adds cards to the free installation success page.

## Relevant technical choices:

*

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

#### Enable feature flag
* If you haven't already, enable the feature flag: add `define( 'YOAST_SEO_INSTALLATION_SUCCESS', true );` to your `wp-config.php` file

#### Test cards
* Navigate to `{admin-url}/admin.php?page=wpseo_installation_successful_free`
* See the cards match the [design](https://www.sketch.com/s/d8601833-a14f-4dff-bee0-426acfa4a2c8/a/098oodO/play).
* See the link opens the configuration workout.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Other environments


## Documentation


## Quality assurance


Fixes [DUPP-138](https://yoast.atlassian.net/browse/DUPP-136)
